mirascope/__init__.py

```diff
@@ -1,6 +1,7 @@
 """mirascope package."""
 import importlib.metadata
 
+from . import integrations
```

Comment:
is this necessary?

Reply:
whoops, left in from trying fixes earlier. removed now